static/app/views/performance/browser/resources/resourceSummaryPage/resourceInfo.tsx

```diff
@@ -31,22 +32,22 @@ function ResourceInfo(props: Props) {
       'On average, this resource is [bytes] when encoded (for example when gzipped).',
       {
         bytes: getDynamicText({
-          value: formatBytesBase2(avgContentLength),
+          value: formatBytesBase2(roundFileSize(avgContentLength)),
           fixed: 'xx KB',
         }),
       }
     ),
     avgDecodedContentLength: tct('On average, this resource is [bytes] when decoded.', {
       bytes: getDynamicText({
-        value: formatBytesBase2(avgDecodedContentLength),
+        value: formatBytesBase2(roundFileSize(avgDecodedContentLength)),
         fixed: 'xx KB',
       }),
     }),
     avgTransferSize: tct(
       'On average, the total bytes transferred over the network (body + headers) for this resource is [bytes].',
       {
         bytes: getDynamicText({
-          value: formatBytesBase2(avgTransferSize),
+          value: formatBytesBase2(roundFileSize(avgTransferSize)),
           fixed: 'xx KB',
```

Comment:
realized because we use base 2, we should switch the fixed values to be kibibyte as well.

```suggestion
          fixed: 'xx KiB',
```

Reply:
👍 Updated!